### DISCUSSION TITLE
Does the record extension has specification?

### DISCUSSION THREAD

**Discussion by molikto** [kriszyp/msgpackr#55] [General]
Spec for msgpack https://github.com/msgpack/msgpack/blob/master/spec.md

**Comment by kriszyp** [comment]
The record extension is documented/specified here:
https://github.com/kriszyp/msgpackr#record-structure-extension-definition

However, if you value a more formal specification/standard, you might want to consider using CBOR (with my [cbor-x](https://github.com/kriszyp/cbor-x) package) instead of MessagePack. CBOR has gone through a full IETF standards track for its [specification](https://www.rfc-editor.org/rfc/rfc8949.html) (MessagePack has not, it is just a spec posted in github), and the corresponding record extension for CBOR has a [complete specification](https://github.com/kriszyp/cbor-records) that has been submitted to the official [CBOR IANA registry](https://www.iana.org/assignments/cbor-tags/cbor-tags.xhtml).
